components/style/core/rtl.less

```diff
@@ -0,0 +1,11 @@
+/* stylelint-disable at-rule-no-unknown */
+
+.rtl {
+  direction: rtl;
+}
+.ltr {
+  direction: ltr;
+}
+.rtl .ant-col {
+  float: right;
+}
```

Comment:
> Do we need global styles now?

These styles will be needed on some cases like direction of `.ant-col` that i'm used.
so every col wrapped in `.rtl` will be displayed correctly. (Used in ConfigProvider RTL demo)